content/articles/jira-web.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Jira Web"
+date: "2020-07-19"
+type: "post"
+authors: ["dittren"]
+tags: [jira]
+lede: "Jira net"
```

Comment:
Add a short summary, or pitch of your post, max about 30 words. This summary will appear on the front page, as well as the Articles page. The thing that helps me write a summary  for my post is the answer to this question, "What is my post about?"